recipes-extend/uenv/uenv_1.0.bb

```diff
@@ -27,7 +27,7 @@ addtask deploy after do_install
 do_deploy () {
     install -d ${DEPLOYDIR}/target/env
     install -m 0644 ${D}/boot/uEnv.txt ${DEPLOYDIR}/target/env
-    install -m 0644 ${D}/boot/Readme.md ${DEPLOYDIR}/target/env
+    install -m 0644 ${D}/boot/Readme.md ${DEPLOYDIR}
```

Comment:
This README.md file explains how to use the uEnv.txt file and needs to be installed in partition 1.

If you deploy it to DEPLOYDIR, ensure you have updated the location at: https://github.com/vudangRVC/meta-renesas-sst/blob/styhead/rz-sbc-support-connectivity/include/core-image-renesas-base.inc#L38

Afterward, clean up this README.md file from the root level.